README.macOS

```diff
@@ -0,0 +1,39 @@
+# This file is a Work in Progress to get FRR compiled
+# and running on MacOS.
+#
+# It is provided as-is without any support. The purpose
+# is to extend the community of users to the MacOS' developers.
+#
+# Should you have any specific comments, please send me
+# direclty an email to vincent.jardin AT 6wind.com
+# or to FRR's mailing list.
+
+# The following compilation has been done with:
+#  - macOS High Sierra (10.13.1)
+#  - xcode (9.1)
+#  - macos ports
```

Comment:
Any success with Home brew instead of mac ports ? Is your proposal is compatible with home brew ? Do you think it is feasible to have a similar documentation letting peoples choice between mac ports and home brew to installed required package for compilation ?

Reply:
Unfortunately, I cannot get deeper on it; I do not have access to a MacOS anymore. I did it for the fun. If you have a MacOS, you are welcome to take over this patch.